### ISSUE TITLE
Viewcube failure after Zoom/ RotateAroundMouseDownPoint

### ISSUE THREAD

**Issue by Sralelb** [helix-toolkit/helix-toolkit#2035]
Hello everyone,

I have a strange behavior after using the Zoom/ RotateAroundMouseDownPoint feature. 
When opening my model and only using the cube to change the view everything works perfect.

Front view only using the cube:
![image](https://github.com/helix-toolkit/helix-toolkit/assets/54818357/bb6bf272-c4bb-4f1e-a23d-b2a41f9fc468)

Front view after using the zoom/rotate feature and when click on the cube:
![image](https://github.com/helix-toolkit/helix-toolkit/assets/54818357/17d77052-41fb-4b5b-9f15-fffc423658b8)

It seems that the center position of the camera has changed. Do I have to set my own camera defautl position to avoid this?

Thanks in advance!

**Comment by holance** [comment]
Yes, it uses camera target to rotate. If your camera target has changed, it will cause this issue.
